package.json

```diff
@@ -10,6 +10,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
```

Comment:
we have used styled component and emotion/styled is also added here. I am curious to know more on this.